client/templates/userprofile/settings.html

```diff
@@ -86,7 +91,7 @@ <h2 class="userprofile-sub-title">
 		<h2 class="userprofile-sub-title">
 				{{ mf 'userProfile-subheading-password' 'Password'}}
 		</h2>
-        <p class="profile_errors">{{perrors}}</p>
+        <p class="password_errors errors">{{perrors}}</p>
```

Comment:
Use spaces around helper names in handlebar helpers. Rename 'pererrors' to 'password_errors' or something more meaningful.